src/main/kotlin/interactor/GetSalaryAverageAndCitiesNamesInCountryInteractor.kt

```diff
@@ -0,0 +1,28 @@
+package interactor
+
+import model.CityEntity
+import java.io.IOException
+
+class GetSalaryAverageAndCitiesNamesInCountryInteractor(
+    private val dataSource: CostOfLivingDataSource
+    ) {
+
+    fun execute(countryName:String):List<Pair<String,Float?>> {
```

Comment:
You have execluded the null salaries, so you have not to make it return **Nalluable** 
```suggestion
    fun execute(countryName:String):List<Pair<String,Float>> {
```
Make it :
`List<Pair<String,**Float**>>`

Reply:
I did that before but i was getting error. I solved it with !! in the second value of pair in the return.
done!